centreon/src/Core/Service/Infrastructure/Repository/DbReadServiceRepository.php

```diff
@@ -555,6 +555,14 @@ private function findServicesRequest(array $accessGroupIds = []): SqlConcatenato
                     LEFT JOIN `:db`.host
                         ON host.host_id = hsr.host_host_id
                         AND host.host_register = '1'
+                    LEFT JOIN `:db`.hostgroup_relation hgr
+                        ON hgr.host_host_id = host.host_id
+                    LEFT JOIN `:db`.hostgroup
+                        ON hostgroup.hg_id = hgr.hostgroup_hg_id
+                    LEFT JOIN `:db`.hostcategories_relation hcr
+                        ON hcr.host_host_id = host.host_id
+                    LEFT JOIN `:db`.hostcategories
+                        ON hostcategories.hc_id = hcr.hostcategories_hc_id
```

Comment:
what about ACLs test on these elements ?

Reply:
And your last join include also host severities

Reply:
Done